docs/suspensive.org/src/pages/docs/react-await/useAwait.ko.mdx

```diff
@@ -0,0 +1,33 @@
+import { Callout } from '@/components'
+
+# Await
+
+<Callout type='experimental'>
+
+`useAwait`는 실험 기능이므로 이 인터페이스는 변경될 수 있습니다.
+
+</Callout>
+
+`useAwait` 받은 비동기 함수가 리턴한 Promise가 pending이면 가장 가까운 부모 Suspense의 fallback을 노출합니다.
+이후 Promise가 fulfilled되면 `useAwait`에서 보장된 awaited data를 사용할 수 있게 합니다.
+
+또한 이 data는 받은 키에 cache되어 재사용하는 경우 pending없이 즉시 사용할 수 있습니다.
+
+```tsx /useAwait/
+import { Await } from '@suspensive/react-await'
+import { Suspense } from '@suspensive/react'
+
+const getPost = (postId: number) => fetch(`/post/${postId}`).then<Post>((res) => res.json())
+
+const AwaitedPost = () => {
+  const awaited = useAwait({ key: ['post', 2] as const, fn: ({ key: [, postId] }) => getPost(postId) })
+
+  return <>{awaited.data}</>
+}
+
+const Example = () => (
+  <Suspense fallback="awaiting...">
+    <AwaitedPost />
+  </Suspense>
+)
+```
```

Comment:
```suggestion
```tsx /useAwait/
import { Await } from '@suspensive/react-await'
import { Suspense } from '@suspensive/react'

const getPost = (postId: number) => fetch(`/post/${postId}`).then<Post>((res) => res.json())

const Post = () => {
  const awaited = useAwait({ key: ['post', 2] as const, fn: ({ key: [, postId] }) => getPost(postId) })

  return <>{awaited.data}</>
}

const Example = () => (
  <Suspense fallback="awaiting...">
    <Post />
  </Suspense>
)
```
```

Reply:
I modified it https://github.com/toss/suspensive/commit/df5377a6e51eba15ec7a8dfb355ce7b043057dbf